extensions/amp-bind/amp-bind.md

```diff
@@ -539,7 +539,6 @@ copyAndSplice(['Pizza', 'Cake', 'Soda'], 2, 1, 'Ice Cream')</pre>
   </tr>
 </table>
 
-
```

Comment:
After merge, check how this looks on the ampproject page: https://www.ampproject.org/docs/reference/components/amp-bind